_layouts/state-page.html

```diff
@@ -264,11 +264,10 @@
         {% assign product_name = product[0] %}
       {% endif %}
       <h3>{{ product_name }}</h3>
-      <p>{{ state_name }} generated
-        {% if revenue %}${{ revenue | intcomma }} in
-        {% else %}no{% endif %}
-        federal revenue from {{ product_name | downcase }} in {{ year }}.
-        Historically:</p>
+      <p>In {{ year }}, 
+        {% if revenue %}companies paid ${{ revenue | intcomma }} to extract{% else %}the federal government did not collect any revenue from extraction of{% endif %} 
```

Comment:
Can these if/else statements be broken up on to multiple lines like above, or does it mess with the spacing?